src/presentation/pages/Home/Main/MainContent.module.scss

```
@@ -0,0 +1,51 @@
+@import '../../../../app/styles/base/settings';
+
+.mainContent {
+  margin: 0 1rem;
+  padding: 1rem;
+  &__item {
+    height: 100%;
+    &TextContainer {
+      margin: 8rem 2rem 1rem 2rem;
+    }
```

Using the margin shorthand this is the same of :


```suggestion
    &TextContainer {
      margin: 8rem 2rem 1rem;
    }
```

Seting 2 rem to horizontal (left and right)